test/guit/fetch.t

```diff
@@ -0,0 +1,18 @@
+Tests fetch command
+  $ mkdir simple
+  $ mkdir simple-clone
+  $ cd simple
+  $ git init
```

Comment:
Recent versions of `git` display a warning wenn calling `git init` without either giving `-b` (which is not available for older git versions) or setting `init.defaultBranch` in the global git configuration.

```suggestion
  $ git init 2> /dev/null
```